mne/io/reference.py

```diff
@@ -97,11 +128,16 @@ def _apply_reference(inst, ref_from, ref_to=None):
                 'using the apply_proj() method function before the desired '
                 'reference can be set.'
             )
+
     for i in projs_to_remove:
         del inst.info['projs'][i]
```

Comment:
For the specific case of N2pc, the topography is weird anyways because the effect is by necessity symmetric:
![](http://www.biologicalpsychiatrycnni.org/cms/attachment/2061065832/2062853858/gr2.jpg)

I think you're right though, it would probably be better to actually remove the non-bipolar channels for the case of the N2pc. (I can't think of another situation where you want to keep the avg and bipolar chans together.)